device/core/src/module_client.ts

```diff
@@ -190,8 +190,8 @@ export class ModuleClient extends InternalClient {
   }
 
   /**
-   * Invokes a method on a downstream device or on another module on the same Edge device. Please note that this feature only works when
-   * the module is being run as part of an Edge device.
+   * Invokes a method on a downstream device or on another module on the same IoTEdge device. Please note that this feature only works when
+   * the module is being run as part of an IoTEdge device.
```

Comment:
I like this fix.  If you search the tree for the " Edge" you might find a few more.